scripts/qa_crowbarsetup.sh

```diff
@@ -2982,15 +2995,18 @@ function custom_configuration()
             fi
 
             if [[ $hacloud = 1 ]] ; then
-                local cinder_volume
                 # fetch one of the compute nodes as cinder_volume
-                cinder_volume=`printf "%s\n" $unclustered_nodes | tail -n 1`
-                # make sure we do not use SP1 node for cinder
+                local cinder_volume=$unclustered_nodes
```

Comment:
This highlights how bad it is that we don't run gating on `automation` for multiple scenarios, only for Cloud 7.